src/devices/bus/multibus/multibus.h

```diff
@@ -87,6 +87,14 @@ class multibus_device
 	template <unsigned I> auto int_callback() { return m_int_cb[I].bind(); }
 	template <unsigned I> void int_w(int state) { m_int_cb[I](state); }
 
+	// wait signal callbacks (split for read/write operations)
+	auto wait_io_rd_cb() { return m_wait_rd_cb.bind(); }
+	auto wait_io_wr_cb() { return m_wait_wr_cb.bind(); }
+
```

Comment:
I object to these callbacks as they don't correspond to the actual Multibus specification. Multibus uses a transfer acknowledge (/XACK) signal to indicate when a read or write transfer is complete (or that a transfer is being timed out). If the iSBC 202 hardware absolutely requires the bus master to be stalled until the slave responds, a more accurate approach to emulating the bus is necessary.

One suggestion is for two-phase bus read/write logic to be implemented in the Multibus bus emulation, preferably also readily supporting hardware that doesn't depend on precise /XACK implementation.